packages/mdc-tabs/tab-bar/mdc-tab-bar.scss

```diff
@@ -47,6 +48,7 @@
 
   // postcss-bem-linter: ignore
   .mdc-toolbar & {
```

Comment:
What does all this code do...the stuff under `.mdc-toolbar &`....which I assume translates to `.mdc-toolbar .mdc-tab-bar`

Why do we need so much code to get a tab bar to work within a toolbar? Maybe we don't need this now that we're customizing color through mixins?

Reply:
We could remove this and then have mixins to customize toolbar specific tab-bars. I like that idea. This will be a breaking change though, so I'll check the rest of the repo.

Reply:
Addressed in https://github.com/material-components/material-components-web/pull/1979/